torchtitan/experiments/deepseek_v3/README.md

```diff
@@ -0,0 +1,39 @@
+# Running DeepSeek in Titan  (experimenatal)
+
+This folders contains a DeepSeek model and kernels and scripts needed to run it.
```

Comment:
These folders contain DeepSeek model support (v2 and v3) with kernels and scripts needed to run it.